packages/web_ui/src/components/InstanceList.tsx

```diff
@@ -86,6 +93,13 @@ export default function InstanceList(props: InstanceListProps) {
 			sorter: (a, b) => strcmp(instancePublicAddress(a), instancePublicAddress(b)),
 			responsive: ["lg"],
 		},
+		{
+			title: "Factorio Version",
+			key: "version",
+			render: (_, instance) => instance.factorioVersion ?? "unknown",
```

Comment:
Here we show the current version read from the server or "unknown". Should we show the configured value (usually "latest") with some icon indicating that it is the configured rather than actual value instead?